vector/src/main/java/im/vector/util/SlashCommandsParser.java

```diff
@@ -95,6 +95,9 @@
     // on / off
     private static final String CMD_MARKDOWN = "/markdown";
 
+    // clear scalar token (waiting for correct 403 management)
+    private static final String CMD_CLEAR_SCALAR_TOKEN = "/clear_scalar_token";
```

Comment:
Is it a command available on riot-web?

Reply:
I don't think so

Reply:
as long there is an issue in riot-meta for that to track parity across Riot clients, I am fine.

Reply:
https://github.com/vector-im/riot-meta/issues/208